integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -3270,29 +3272,26 @@ public static String getIntrospectJobName(String domainUid) {
    * @param domain custom resource object
    */
   public static synchronized void setPodAntiAffinity(Domain domain) {
-    V1PodAntiAffinity podAntiAffinity = new V1PodAntiAffinity()
-        .addPreferredDuringSchedulingIgnoredDuringExecutionItem(
-            new V1WeightedPodAffinityTerm()
-                .weight(100)
-                .podAffinityTerm(new V1PodAffinityTerm()
-                    .topologyKey("kubernetes.io/hostname")
-                    .labelSelector(new V1LabelSelector()
-                        .addMatchExpressionsItem(new V1LabelSelectorRequirement()
-                            .key("weblogic.clusterName")
-                            .operator("In")
-                            .addValuesItem("($CLUSTER_NAME)")))
```

Comment:
Just FYI... The bug in this is that you have "($CLUSTER_NAME)" instead of "$(CLUSTER_NAME)"

Reply:
corrected and verified in https://build.weblogick8s.org:8443/job/wko-oke-nightly-seqential/34